Added link in Triage to go to medical page for open encounters.

AJSclyn opened the pull request:
When searching for a patient that has an open encounter in the Triage page, the page will display a link to medical saying that patient has an open encounter.

https://teamfemr.atlassian.net/browse/FEMR-22
https://teamfemr.atlassian.net/browse/FEMR-103
https://teamfemr.atlassian.net/browse/FEMR-104

kdunlap commented on app/femr/ui/views/triage/index.scala.html:
```diff
@@ -26,6 +26,10 @@
         }
+        @if(viewModel.isLinkToMedical()){
+
+           <p> <a class="btn btn-danger" href="/medical/edit/@viewModel.getPatient().getId()">Patient has an open encounter!</a></p>
```
Try something more like this. You may need some css still. The button might need

`display: inline-block;`

`<p>Patient has an open encounter <a class="btn btn-danger" href="/medical/edit/@viewModel.getPatient().getId()">Go To Medical</a></p>`

AJSclyn commented:
Apparently I didn't have to edit the css, and the web page scales nicely with the button/warning.  Just changed the link and added a warning to the side.

![image](https://cloud.githubusercontent.com/assets/10897508/11605947/7e049fb6-9adc-11e5-8795-b311be60ae7e.png)

kevinzurek commented on app/femr/ui/controllers/TriageController.java:
```diff
@@ -125,6 +132,13 @@ public Result indexPopulatedGet(int patientId) {
         viewModelGet.setVitalNames(vitalServiceResponse.getResponseObject());
         viewModelGet.setPossibleAgeClassifications(patientAgeClassificationsResponse.getResponseObject());
+        //Patient has an open encounter for medical
+        if(patientEncounter.getMedicalDateOfVisit() == null){
```
AJ,

An encounter also has a time limit of 2 days set to close if the medical date of visit is null. Try 'patientEncounter.getIsClosed()' instead of 'patientEncounter.getMedicalDateOfVisit()'. :)

The logic for a closed encounter can be found here:

https://github.com/FEMR/femr/blob/8ce6459eb8828d95855ec72e635cc56eaf462121/app/femr/business/helpers/LogicDoer.java#L98

-Kevin

kevinzurek commented:
Looks good to me, I think AJ wins most descriptive commit messages
